src/renderer/app.js

```diff
@@ -36,9 +36,12 @@ if (isDev) {
 const initialState = getInitialStateRenderer();
 const store = configureStore(initialState, 'renderer');
 
+if (store.getState().preferences.language === undefined) {
+	store.getState().preferences.language = navigator.language;
```

Comment:
Thank you for the Pull Request! Although the implementation intent is correct, there are issues with the proposed code changes. Here for example, the returned store state is meant to be immutable (i.e. should only be modified via `setState()` so that other components can react asynchronously). There are other lifecycle issues in this PR. I will propose an alternative approach in a separate PR, taking into account Electron's own system language (not just Chromium's `navigator.language`).

Reply:
Thanks for your work. For now, I can't use setState since I get errors. I will wait for your PR for this.